src/shop-app.html

```diff
@@ -242,21 +234,18 @@
 
     <app-header role="navigation" id="header" effects="waterfall" condenses reveals>
       <app-toolbar>
-        <div class="left-bar-item">
-          <paper-icon-button
-              class="menu-btn"
-              icon="menu"
-              on-tap="_toggleDrawer"
-              aria-label="Categories">
-          </paper-icon-button>
-          <a class="back-btn" href="/list/[[categoryName]]" tabindex="-1">
-            <paper-icon-button icon="arrow-back" aria-label="Go back"></paper-icon-button>
+        <div class="icon-button-container">
+          <a href="#" class="shop-icon-button menu-btn" on-click="_toggleDrawer" aria-label="Categories">
```

Comment:
@frankiefu I think app-route installs a global click listener and calls `preventDefault()`, but why is `href="#"` needed?